src/test/tests/expressions/mesh_quality.py

```diff
@@ -0,0 +1,48 @@
+# ----------------------------------------------------------------------------
+#  CLASSES: nightly
```

Comment:
There already exists hybrid/mesh_quality.py, is there a reason you didn't add your tests there?

Reply:
Honestly, I've never fully understood the hybrid section and what it's used for. I assumed that it was for "hybrid" tests that combine many different features of VisIt, but there are tests in there that don't quite follow that idea. 

I can move this test to the hybrid/mesh_quality file if that makes more sense.

Reply:
It seems better to keep all the mesh_quality tests together.
Hybrid may need a reorganization (eg 'expressions' in that folder, perhaps should be in 'expressions' folder)? But that's a discussion for a different ticket.

Reply:
Done. I'll open an issue referencing this conversation.

Reply:
Thanks for opening the new issue.